src/model/version.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Entity, PrimaryKey, Property, ManyToOne } from '@mikro-orm/core';
+import { Package } from './package.entity';
+
+@Entity()
+export class Version {
+  @PrimaryKey()
+  id!: number;
+
+  @ManyToOne(() => Package)
+  package!: Package;
+
+  @Property()
+  major!: number;
+
+  @Property()
+  minor!: number;
+
+  @Property()
+  patch!: number;
+
+  @Property({ unique: true })
+  sortableVersion!: string;
```

Comment:
This approach will not work. Look on this:
![image](https://github.com/user-attachments/assets/c5031639-61fd-4bd3-a313-6ddc9bffc796)

Reply:
Ok I have another thought on this: let's remove all of sorting on top of DB level. If needed we will add later. We don't need it anyway now, since we don't support `latest`. In every case we return to the users all versions, also we don't expect this "versions" load to be very big now. So pls, remove `major`, `minor`, `patch` from DB. Leave only 1 field: `version` string (composite primary key with package_name)